solidity/utils/test_deployment.js

```diff
@@ -169,10 +169,13 @@ const run = async () => {
     const converterRegistryData = await web3Func(deploy, 'converterRegistryData', 'ConverterRegistryData', [
         contractRegistry._address
     ]);
+
+    const networkFeeWallet = await web3Func(deploy, 'networkFeeWallet', 'TokenHolder', []);
     const networkSettings = await web3Func(deploy, 'networkSettings', 'NetworkSettings', [
-        getConfig().networkSettingsParams.networkFeeWallet,
-        percentageToPPM(getConfig().networkSettingsParams.networkFee)
+        networkFeeWallet._address,
```

Comment:
If you choose to remove these configuration parameters, then you also need to update the README.md file (same folder).

Reply:
With your permission, I'll remove it from `README.md` as well. It's reconfigure, so it's unnecessary to configure it with a value during a test deployment and the less configuration options we'd have - the better